tests/pc/test_po_update.py

```diff
@@ -105,7 +113,9 @@ def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
         time.sleep(30)
         int_facts = asichost.interface_facts()['ansible_facts']
         pytest_assert(not int_facts['ansible_interface_facts'][portchannel]['link'])
-        pytest_assert(wait_until(120, 10, 0, asichost.check_bgp_statistic, 'ipv4_idle', 1))
+        pytest_assert(
+            bgp_enabled(duthost) and wait_until(120, 10, 0, asichost.check_bgp_statistic, 'ipv4_idle', 1)
+            or not pc_active(asichost, portchannel))
```

Comment:
Probably need to check pc_active using wait_util logic too?

Reply:
Fixed, invoke pc_active using wait_util